src/test/e2e/25_helm_test.go

```diff
@@ -55,7 +55,23 @@ func testHelmChartsExample(t *testing.T) {
 	require.Contains(t, e2e.StripMessageFormatting(stdErr), "chart \"asdf\" version \"6.4.0\" not found")
 	require.Contains(t, e2e.StripMessageFormatting(stdErr), "Available charts and versions from \"https://stefanprodan.github.io/podinfo\":")
 
-	// Create the package (with a registry override to test that as well)
+	// Create a test package (with a registry override (host+subpath to host+subpath) to test that as well)
+	stdOut, stdErr, err = e2e.Zarf("package", "create", "examples/helm-charts", "-o", "build", "--registry-override", "ghcr.io/stefanprodan=docker.io/stefanprodan", "--tmpdir", tmpdir, "--confirm")
+	require.NoError(t, err, stdOut, stdErr)
+
+	// Create a test package (with a registry override (host to host+subpath) to test that as well)
+	// expect to fail as ghcr.io is overriden and the expected final image doesn't exist but the override works well based on the error message in the output
+	stdOut, stdErr, err = e2e.Zarf("package", "create", "examples/helm-charts", "-o", "build", "--registry-override", "ghcr.io=localhost:555/noway", "--tmpdir", tmpdir, "--confirm")
+	require.Error(t, err, stdOut, stdErr)
+	require.Contains(t, string(stdErr), "localhost:555")
```

Comment:
```suggestion
	require.Contains(t, string(stdErr), "localhost:555/noway")
```
This should check for localhost:555/noway to make sure the /noway path is really getting set